scripts/examples/gn_efr32_example.sh

```diff
@@ -22,13 +22,22 @@ set -e
 
 source "$(dirname "$0")/../../scripts/activate.sh"
 
+sdk_root="$PWD/third_party/efr32_sdk/repo"
```

Comment:
Please don't use PWD in our shell scripts as this will break them when running in other directories.

Set and use CHIP_ROOT instead.